pallets/asset/src/lib.rs

```diff
@@ -1695,9 +1725,12 @@ impl<T: Trait> Module<T> {
         Ok(())
     }
 
-    /// Get the asset `id` balance of `who`.
-    pub fn balance(ticker: &Ticker, did: &IdentityId) -> T::Balance {
-        Self::balance_of(ticker, did)
+    /// Get the asset `ticker` balance of `did`, both total and that of the default portfolio.
+    pub fn balance(ticker: &Ticker, did: IdentityId) -> (T::Balance, T::Balance) {
```

Comment:
is it possible to provide the name to the return variable as it makes hard to read the code with 0's and 1's ?

Reply:
I'll add a temporary struct. The reason I didn't perfect this aspect is because the default portfolio plays a distinguished role only until we implement the ability to use any portfolio as the sender. Then this function may revert to just returning the total balance.